features/orders.feature

```diff
@@ -1272,25 +1408,3 @@ Feature: Orders
       }
       """
 
-    Scenario: Get order timing
```

Comment:
Why is it removed?

Reply:
This test is not actually testing anything. We've added this test at the begining of work with mercadopago to get the response from preferences endpoint. But now id doesn't work because we would need to set an Access Token to get a preference and we don't have that for now.

Reply:
Ok 👍